pkg/cluster/internal/create/actions/waitforready/waitforready.go

```diff
@@ -69,13 +69,13 @@ func (a *Action) Execute(ctx *actions.ActionContext) error {
 	isReady := waitForReady(node, startTime.Add(a.waitTime))
 	if !isReady {
 		ctx.Status.End(false)
-		fmt.Println(" • WARNING: Timed out waiting for Ready ⚠️")
+		ctx.Logger.V(0).Info(" • WARNING: Timed out waiting for Ready ⚠️\n")
```

Comment:
For consistency with other log statements we should maybe drop the trailing newline, the logger ensures a trailing newline.

Same for below

Reply:
addressed.